frontend/app/src/store/defi/protocols/index.ts

```diff
@@ -150,7 +144,7 @@ export const useDefiSupportedProtocolsStore = defineStore(
                 timestamp: event.timestamp,
                 txHash: event.txHash,
                 extras: {}
-              } as DefiLendingHistory<typeof DefiProtocol.AAVE>;
+              } satisfies DefiLendingHistory<typeof DefiProtocol.AAVE>;
```

Comment:
It is a new operator in 4.9 :) I think in most of these cases it makes more sense since it will enforce that the object follows the type/schema but it will not hide any errors.